skimage/util/dtype.py

```diff
@@ -39,7 +39,11 @@ def dtype_limits(image, clip_negative=True):
     clip_negative : bool, optional
         If True, clip the negative range (i.e. return 0 for min intensity)
         even if the image dtype allows negative values.
+        The default behavior (None) is equivalent to True.
     """
+    if clip_negative is None:
+        clip_negative = True
+        warn('The default will change to `False` in version 0.15')
```

Comment:
This needs to be more specific, because most users will see this warning when calling a higher level function. "The default value of `clip_negative` will change to `False` in version 0.15." (I believe that the function name will appear with the warning but I'm not sure — if not, then include the function name also.